src/common/libjob/specutil.c

```diff
@@ -38,10 +39,13 @@ json_t *specutil_argv_create (int argc, char **argv)
     return NULL;
 }
```

Comment:
My bad, but errno handling is not right here, since jansson functions do not set errno.

It should be something like this instead:
```c
json_t *specutil_argv_create (int argc, char **argv)
{
    int i;
    json_t *a, *o;

    if (!(a = json_array ()))
        goto nomem;
    for (i = 0; i < argc; i++) {
        if (!(o = json_string (argv[i])) || json_array_append_new (a, o) < 0) {
            json_decref (o);
            goto nomem;
        }
    }
    return a;
nomem:
    json_decref (a);
    errno = ENOMEM;
    return NULL;
}
```